pkg/ccl/changefeedccl/kvfeed/kv_feed.go

```diff
@@ -108,7 +108,7 @@ func Run(ctx context.Context, cfg Config) error {
 	// down the changefeed exposes synchronization challenges.
 	var scErr schemaChangeDetectedError
 	if errors.As(err, &scErr) {
-		log.Infof(ctx, "stopping changefeed due to schema change at %v", scErr.ts)
+		log.Infof(ctx, "stopping kv feed due to schema change at %v", scErr.ts)
```

Comment:
do you know why we're just not returning an error here?

Reply:
I do not.  I've read through the code and I believe that it would _currently_ be OK to return early from here because the unbuffered channel we use is already acting as a synchronisation point. I've removed the <-ctx.Done() and didn't see any test failures, but I want to run that under stress a bit more just in case I missed something in my read through.